src/util/actor.js

```diff
@@ -123,6 +124,8 @@ class Actor {
     }
 
     processTask(id: number, task: any) {
+        // Always delete since we are no longer cancellable
+        delete this.cancelCallbacks[id];
```

Comment:
Nit: if we're doing this here, we can probably remove the same line a few lines down.

Reply:
Agreed, thanks for the catch.